src/components/Protein/Summary/index.tsx

```diff
@@ -0,0 +1,336 @@
+                  </td>
+                  <td>
+                    <DescriptionReadMore
+                      text={metadata.description[0]}
+                      minNumberOfCharToShow={250}
+                      patternToRemove="\s?\(PubMed:\d+\)\s?"
+                    />
+                  </td>
+                </tr>
+              ) : null}
+              <tr>
+                <td>Family membership</td>
+                <td>
+                  {families?.length ? (
+                    //@ts-ignore
+                    <ProteinEntryHierarchy entries={families} />
+                  ) : (
+                    <div className={css('margin-bottom-medium')}>
+                      {matchesLoaded ? 'None predicted' : <Loading inline />}
+                    </div>
+                  )}
+                </td>
+              </tr>
+
+              {metadata?.counters?.isoforms &&
+              (metadata.counters.isoforms as number) > 0 ? (
+                <tr>
+                  <td data-testid="protein-function">
+                    Isoforms{' '}
+                    <Tooltip
+                      title={`This protein has ${metadata.counters.isoforms} isoforms`}
+                    >
+                      <span
+                        className={css('small', 'icon', 'icon-common')}
+                        data-icon="&#xf129;"
+                        aria-label={`This protein has ${metadata.counters.isoforms} isoforms`}
+                      />
+                    </Tooltip>
+                  </td>
+                  <td style={{ display: 'flex' }}>
+                    <IsoformSelector />
+                    {renderComparisonButton ? (
+                      <FullScreenButton
+                        element={comparisonContainerRef.current}
+                        className={css(
+                          'vf-button',
+                          'vf-button--secondary',
+                          'vf-button--sm',
+                          'comparison-button',
+                          'icon',
+                          'icon-common',
+                          { disabled: !isoform }
+                        )}
+                        disabled={!isoform}
+                        dataIcon={'\uF0DB'}
+                        tooltip="View in comparison mode"
+                        onFullScreenHook={() => setComparisonMode(true)}
+                        onExitFullScreenHook={() => setComparisonMode(false)}
+                      />
+                    ) : null}
+                  </td>
+                </tr>
+              ) : null}
+            </tbody>
+          </table>
+        </div>
+        <div className={css('vf-stack')}>
+          <section>
+            <section>
+              <h5>External Links</h5>
+              <ul className={css('no-bullet')}>
+                <li>
+                  <UniProtLink id={metadata.accession} className={css('ext')}>
+                    View {metadata.accession} in UniProtKB
```

Comment:
```suggestion
                    UniProt
```

(Technically, the service is UniProt. UniprotKB is the database of proteins.)